xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.35.xml

```diff
@@ -25,7 +25,7 @@
                <msContents>
                   <textLang mainLang="grc" otherLangs=""/>
                   <msItem class="#funerary">
```

Comment:
If you're going to call it an epitaph in the description, make sure to make the control vocab reflect that.

Reply:
Should I be tagging it as #funerary.epitath ?